discovery-provider/alembic/versions/5aaa6198def8_user_primary_secondary_columns.py

```diff
@@ -0,0 +1,30 @@
+"""user primary secondary columns
+
+Revision ID: 5aaa6198def8
+Revises: 6d1b38f242fe
+Create Date: 2020-12-22 13:00:09.925084
+
+"""
+from alembic import op
+import sqlalchemy as sa
+
+
+# revision identifiers, used by Alembic.
+revision = '5aaa6198def8'
+down_revision = '579360c7cbf3'
+branch_labels = None
+depends_on = None
+
+
+def upgrade():
+    # ### commands auto generated by Alembic - please adjust! ###
+    op.add_column('users', sa.Column('primary_id', sa.Integer(), nullable=True))
```

Comment:
per discussion with @SidSethi adding another field replica_set_update_signer to also index the new _signer event field. this will be useful for us in the case of diagnosing any potential incorrect updates

Reply:
added